recipe/meta.yaml

```diff
@@ -35,7 +35,7 @@ requirements:
     - pystan >=2.14
     # Pystan models depend on numpy and as they are compiled and pickled we
     # need to pin numpy version.
-    - numpy 1.10.*    
+    - numpy 1.11.*
 
   run:
     - python
```

Comment:
Can you add `numpy >=1.11` here? Also changing numpy to `1.10` in both build and run works for me. (When numpy is not in run for 1.10, it fails like you mentioned)

Reply:
I think the idea is to use the minimum numpy version at build time so the pickled objects have forward compatibility. When using `1.10` the problem is that `conda-smithy rerender` removes all CI files. See https://github.com/conda-forge/conda-smithy/issues/601

Reply:
It works correctly if you have
```
   build:
      - numpy 1.10.*
   run:
      - numpy >=1.10
```

You only had
```
   build:
      - numpy 1.10.*
```

Reply:
@isuruf you are right. Do you mind to chime in https://github.com/conda-forge/conda-smithy/issues/601 ?